services/README.md

```diff
@@ -0,0 +1,51 @@
+# Guidelines for using self-hosted services in MetaBrainz projects
+
+This is a set of guidelines for the main contributors of MetaBrainz
+projects to keep their use of self-hosted services maintainable.
+
+This document mentions general guidelines only about **documentation
+requirements** for a project when using of a self-hosted service so
```

Comment:
"of" seems wrong here, just "when using a self-hosted service" I guess? :)

Reply:
Correct, fixed in https://github.com/metabrainz/guidelines/pull/17/commits/e1d5fcb5665cc986dafb4357e47348cd5850c3f5